app/components/DonutChart/Path.js

```diff
@@ -28,9 +28,9 @@ class Path extends React.Component<PathProps> {
       .attrTween('d', () => t => arcFn(interpolateArc(t)));
   }
 
-  pathRef: ?HTMLElement;
+  pathRef: ?React.ElementRef<any>;
```

Comment:
also included updates for yarn run flow / npm run flow so that we don't have any more error when running that command